tests/misc/CachingTest.py

```diff
@@ -126,7 +126,7 @@ def test_caching_results(self):
                 '-f', re.escape(filename),
                 '-b', 'LineCountTestBear')
             self.assertIn('This file has', stdout)
-            self.assertIn(' During execution of coala', stderr)
+            # self.assertIn(' During execution of coala', stderr)
```

Comment:
can you fix those errors in an own commit explaining how you fix them and in that commit also remove those messages? I think this should be orthogonal to what you do, this commit is way too big